src/components/TodoItem.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  selectedTodo: number | null;
+  clearCompleted: boolean;
+  handleDeleteTodo: (todoId: number) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  selectedTodo,
+  clearCompleted,
+  handleDeleteTodo,
+}) => {
+  return (
+    <div
+      data-cy="Todo"
+      className={classNames('todo', { completed: todo.completed })}
+      key={todo.id}
+      // onDoubleClick={() => handleEditTodo(todo.id)}
+    >
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={todo.completed}
+          // onChange={() => handleChangeStatus(todo.id)}
+        />
+      </label>
+
+      <span data-cy="TodoTitle" className="todo__title">
+        {todo.title}
```

Comment:
Use destructuring for `todo`

Reply:
i will fix it in 3 part of task ok? becouse i steel in progres